### DISCUSSION TITLE
change to be made on importing

### DISCUSSION THREAD

**Discussion by mk-ling** [Nikeshbajaj/spkit#7] [Ideas]
Hi,

First of all, thx for the contribution, I appreciate it. However, when I tried to import spkit from my project path(which also have a package named "core"), the import statements in spkit(eg. from core import xxx) locates to the "core" package in my project instead of the one in spkit. So I'd suggest to use relative import in such places that might have ambiguities(just like what numpy did if you look at their code). Other than this, spkit has been very helpful. Again, thx for the contribution from you guys.

**Comment by Nikeshbajaj** [comment]
Thanks @mk-ling for pointing out. I found similar issue with other names. This will be fixed in next update. 

Assign to @Nikeshbajaj

**Reply by mk-ling** [reply]
Appreciate it!

**Comment by Nikeshbajaj** [comment]
It is fixed in new release 0.0.9.5 ` https://pypi.org/project/spkit/`

**Reply by mk-ling** [reply]
Thanks for the update! I will sync to the latest release soon. Also, I have sited the project. Hopefully it gets more people to use it, as it's one of the best open-source projects for signal processing imo yet not getting enough attention. Respect for your work!

**Reply by Nikeshbajaj** [reply]
Thanks @mk-ling Appreciate!!! 👍
